boards/nordic/nrf52dk/src/main.rs

```diff
@@ -135,20 +135,20 @@ pub unsafe fn reset_handler() {
         components::gpio_component_helper!(
             nrf52832::gpio::GPIOPin,
             // Bottom right header on DK board
-            &nrf52832::gpio::PORT[Pin::P0_03],
-            &nrf52832::gpio::PORT[Pin::P0_04],
-            &nrf52832::gpio::PORT[Pin::P0_28],
-            &nrf52832::gpio::PORT[Pin::P0_29],
-            &nrf52832::gpio::PORT[Pin::P0_30],
-            &nrf52832::gpio::PORT[Pin::P0_31],
+            3 => &nrf52832::gpio::PORT[Pin::P0_03],
+            4 => &nrf52832::gpio::PORT[Pin::P0_04],
+            28 => &nrf52832::gpio::PORT[Pin::P0_28],
+            29 => &nrf52832::gpio::PORT[Pin::P0_29],
+            30 => &nrf52832::gpio::PORT[Pin::P0_30],
+            31 => &nrf52832::gpio::PORT[Pin::P0_31],
             // Top mid header on DK board
-            &nrf52832::gpio::PORT[Pin::P0_12],
-            &nrf52832::gpio::PORT[Pin::P0_11],
+            12 => &nrf52832::gpio::PORT[Pin::P0_12],
+            11 => &nrf52832::gpio::PORT[Pin::P0_11],
             // Top left header on DK board
-            &nrf52832::gpio::PORT[Pin::P0_27],
-            &nrf52832::gpio::PORT[Pin::P0_26],
-            &nrf52832::gpio::PORT[Pin::P0_02],
-            &nrf52832::gpio::PORT[Pin::P0_25]
+            27 => &nrf52832::gpio::PORT[Pin::P0_27],
+            26 => &nrf52832::gpio::PORT[Pin::P0_26],
+            2 => &nrf52832::gpio::PORT[Pin::P0_02],
+            25 => &nrf52832::gpio::PORT[Pin::P0_25]
```

Comment:
This still looks like a breaking change for the nrf52dk?

Reply:
Sorry, somehow this slipped, fixing it now.